.npmignore

```diff
@@ -3,7 +3,8 @@
 /scripts
 /jest.config.js
 /tsconfig.*
+/test
+
 *.tsbuildinfo
-*.test.*
 *.snap
```

Comment:
You should be able to drop this line now too.

Reply:
Done